CONTRIBUTING.md

```diff
@@ -176,3 +145,32 @@ package.
   * Do all the Travis CI and CircleCI tests pass?
   * If applicable, has an entry been added into the changelog?
   * Can you check out the pull request and repeat the examples and tests?
+
+Other Tips
+----------
+
+- To prevent the automated tests from running, you can add ``[ci skip]`` to your
+  commit message. This is useful if your PR is a work in progress and you are
+  not yet ready for the tests to run. For example:
+
+      $ git commit -m "WIP widget [ci skip]"
+
+  - If you already made the commit without including this string, you can edit
+    your existing commit message by running:
+
+        $ git commit --amend
+
+- To skip only the tests running on Travis CI use ``[skip travis]``.
+
+- If your commit makes substantial changes to the documentation but no code
```

Comment:
Maybe combine this and the last point in a shortened one, and call it 'skip ci'?

Reply:
It really seems like this whole section has a lot of redundancy, and can be reduced to a single bullet-point about skip CI.

Reply:
see 36ec924

Reply:
Oops, scratch that.  Just realized that section has already been rewritten in main, so I'm just rebasing and keeping the existing text that's there.